### ISSUE TITLE
Add ability to revert only after X failed builds.

### ISSUE THREAD

**Issue by yogurtearl** [ki82/svn-revert-plugin#1]
Some times tests are slightly flaky and we only want to revert if the tests fails X number of time in a row.

**Comment by davidparsson** [comment]
That's a nice suggestion, but I believe that it's rather tricky to implement the way we've solved this today. Since we're not developing plugin that actively, we'll be happy to accept pull requests.

**Comment by davidparsson** [comment]
How would you have a test run several times? Should a test be re-run in a single build, should a specific revision have several builds if a test fails, or should the plugin revert a previous revision when building a later? Either solution has its own problems.

**Comment by davidparsson** [comment]
Won't fix. Pull requests are however welcome.
